tests/python/pants_test/task/test_goal_options_mixin_integration.py

```diff
@@ -43,21 +43,42 @@ def get_echo(which):
             return None
           else:
             with open(path, 'r') as fp:
-              return [os.path.basename(x.strip()) for x in fp.readlines()]
-        self.assertEqual(expected_one, get_echo('one'))
-        self.assertEqual(expected_two, get_echo('two'))
+              return {os.path.basename(x.strip()) for x in fp.readlines()}
+
+        self.assertEqual(expected_one, get_echo('one'),
+                         'one expected {}, but got {}'.format(expected_one, get_echo('one')))
```

Comment:
Why add these messages? `assertEqual` is supposed to already generate a sensible message.